cmd/zetatool/config/config.go

```diff
@@ -0,0 +1,81 @@
+package config
+
+import (
+	"encoding/json"
+
+	"github.com/spf13/afero"
+)
+
+var AppFs = afero.NewOsFs()
+
+const (
+	Flag                      = "config"
+	defaultCfgFileName        = "InboundTxFilter_config.json"
+	ZetaURL                   = "http://46.4.15.110:1317" //http://100.71.167.102:26657
+	TssAddressBTC             = "bc1qm24wp577nk8aacckv8np465z3dvmu7ry45el6y"
+	TssAddressEVM             = "0x70e967acfcc17c3941e87562161406d41676fd83"
+	BtcExplorer               = "https://blockstream.info/api/address/bc1qm24wp577nk8aacckv8np465z3dvmu7ry45el6y/txs"
+	EthRPC                    = "https://rpc.ankr.com/eth/2da24e4a1fd28f2bec1569eceb2c38a5694b7f5c83fd24c69ae714a89a514f9b"
+	ConnectorAddress          = "0x000007Cf399229b2f5A4D043F20E90C9C98B7C6a"
+	CustodyAddress            = "0x0000030Ec64DF25301d8414eE5a29588C4B0dE10"
+	EvmStartBlock      uint64 = 19200110
+	EvmMaxRange        uint64 = 1000
+)
+
+type Config struct {
```

Comment:
Need comment on the structure. What config it is for.
Same for the field unless the name is clear.
What BtcExplorer represent? url? API key?